aiohttp/web_protocol.py

```diff
@@ -523,8 +537,6 @@ async def start(self) -> None:
                     # wait for next request
                     self._waiter = loop.create_future()
                     await self._waiter
-                except asyncio.CancelledError:
-                    break
```

Comment:
Similar story here, it's start() is created as a task, which gets cancelled explicitly. Not sure why this looks like a public method, but I doubt anyway is dealing with this directly. But, again, the only code awaiting on the task is already handling CancelledError.